src/pages/home/sidebar/ChatLinkRow.js

```diff
@@ -63,7 +62,6 @@ const ChatLinkRow = ({
         >
             <PressableLink
```

Comment:
Since you're getting rid of the `to` prop, it seems like we no longer need this to be a `Link` (or our custom `<PressableLink>`). Can we just use a `<Pressable/>` or `<TouchableOpacity/>` instead?

Reply:
Yeah. That's a good point... I'm not sure we even need this component at all it doesn't seem like it serves a clear purpose.